hack/vendor.sh

```diff
@@ -55,7 +55,7 @@ clone hg code.google.com/p/go.net 84a4013f96e0
 clone hg code.google.com/p/gosqlite 74691fb6f837
 
 #get libnetwork packages
-clone git github.com/docker/libnetwork 005bc475ee49a36ef2ad9c112d1b5ccdaba277d4
+clone git github.com/docker/libnetwork f72ad20491e8c46d9664da3f32a0eddb301e7c8d
```

Comment:
@runcom I If this commit would address only the docker side part, issue would not be fixed.
I think it would not be correct to have a commit in the tree which is only a partial fix.
I could see a case where a maintainer has to cherry-pick fixes from a master branch to a release branch: It would not be fair he had to do the extra work of correlating an issue to multiple commits. It would also be an error prone process.